cabal-install/cabal-install.cabal

```diff
@@ -63,6 +63,8 @@ library
     default-extensions: TypeOperators
 
     hs-source-dirs:   src
+    other-modules:
+        PackageInfo_cabal_install
```

Comment:
How does this interact with the recent `Cabal-version` issue?

Reply:
This is meant to solve the issue at the root, by automating the generation of this constant

Reply:
I think @geekosaur means https://github.com/haskell/cabal/issues/9331.

Reply:
Yes, that.

Reply:
Ah this one, indeed. I have switched to using `Paths_cabal_install`.